public/components/Flyout/flyouts/createDetector.js

```diff
@@ -0,0 +1,814 @@
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import React, { Component, Fragment, useState, useEffect } from 'react';
+import { Formik } from 'formik';
+import { hasError, isInvalid, required, validatePositiveInteger } from '../../../utils/validate';
+
+import {
+  EuiFlexItem,
+  EuiText,
+  EuiFormRow,
+  EuiButton,
+  EuiPageBody,
+  EuiSpacer,
+  EuiBasicTable,
+  EuiTitle,
+  EuiFlyoutFooter,
+  EuiFlexGroup,
+  EuiButtonEmpty,
+  EuiFlyoutBody,
+  EuiCallOut,
+  EuiIcon,
+  EuiTextColor,
+  EuiPanel,
+  EuiFlexGrid,
+  EuiFormLabel,
+  EuiLink,
+  EuiPopover,
+  EuiExpression,
+} from '@elastic/eui';
+import ContentPanel from '../../ContentPanel';
+import {
+  FormikFieldText,
+  FormikFieldNumber,
+  FormikComboBox,
+  FormikSelect,
+} from '../../FormControls';
+import { KIBANA_AD_PLUGIN, DATA_TYPES, DETECTOR_CREATION_CALLOUTS } from '../../../utils/constants';
+import { getIndexFields } from '../../../pages/CreateMonitor/components/MonitorExpressions/expressions/utils/dataTypes';
+import {
+  getOperators,
+  isNullOperator,
+  isRangeOperator,
+  validateRange,
+  displayText,
+} from '../../../pages/CreateMonitor/components/MonitorExpressions/expressions/utils/whereHelpers';
+import {
+  POPOVER_STYLE,
+  EXPRESSION_STYLE,
+  WHERE_BOOLEAN_FILTERS,
+} from '../../../pages/CreateMonitor/components/MonitorExpressions/expressions/utils/constants';
+import { getPathsPerDataType } from '../../../pages/CreateMonitor/containers/DefineMonitor/utils/mappings';
+import { formikToWhereClause } from '../../../pages/CreateMonitor/containers/CreateMonitor/utils/formikToMonitor';
+import { MAX_INTERVAl_LENGTH_MINUTES } from '../../../pages/MonitorDetails/containers/utils/helpers';
+
+function toString(obj) {
+  if (typeof obj != 'undefined') {
+    if (obj.hasOwnProperty('period')) {
+      let period = obj.period;
+      return period.interval;
+    } else if (typeof obj == 'number') {
+      // epoch
+      return moment(obj).format('MM/DD/YY hh:mm A');
+    }
+  }
+  return '-';
+}
+
+class FeaturePreview extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    const featureAttributes = this.props.featureAttributes;
+    const items = [
+      {
+        name: featureAttributes.feature_name,
+        definition: '',
+        state: 'Enabled',
+      },
+    ];
+    const columns = [
+      {
+        field: 'name',
+        name: 'Feature name',
+      },
+      {
+        field: 'definition',
+        name: 'Feature definition',
+        render: () => {
+          return (
+            <div>
+              <p>
+                {' '}
+                <b>Field:</b> {featureAttributes.fieldName || ''}
+              </p>
+              <p>
+                {' '}
+                <b>Aggregation method:</b> {featureAttributes.aggregationType || ''}
```

Comment:
Why don't we show "Field" and "Aggregation method" as separate columns, like "Feature name" and "State"?

Reply:
This was to keep the same style displayed in the configurations page on the anomaly detection plugin since its the same format that the AD plugin follows. I will discuss this further with UX.